src/terra-dev-site/guide/CreatingItemCollection.f.guide.mdx

```diff
@@ -0,0 +1,187 @@
+import ResponsiveListandTableExample from '../example/ResponsiveListandTableExample?dev-site-example';
+import ResponsiveListandCellGridExample from '../example/ResponsiveListandCellGridExample?dev-site-example';
+import ResponsiveListandHtmlTableExample from '../example/ResponsiveListandHtmlTableExample?dev-site-example';
+
+# Creating Item Collection
+
+Start by making a `ResponsiveElement` that will switch between its two sub-components, a `Table` version of the content when above a certain size and a `List` version of the same content as the default display. In this `ResponsiveElement`, we set 3 attributes. The first is the defaultElement which is just the `List` version of the content. The second is the `responsiveTo` attribute, which determines what the component compares itself to in order to resizes itself, its parent or the window. Terra's `ItemCollection` surrounds its `ResponsiveElement` in a `<div>` tag, and sets `responsiveTo` to `parent`. Finally the breakpoint you want to switch at is defined by setting one of the following attributes: `tiny`, `small`, `medium`, `large`, or `huge` to the `Table` version of the content.
```

Comment:
Can we add links to all components that are referenced?

Question, should we recommend creating with a responsive element or flip on a terra-application/breakpoint value themselves like:
```js
import React, { useState, useRef } from 'react';
import { ActiveBreakpointContext } from 'orion-application/lib/breakpoints';

const Example = (props) => {
  const activeBreakpoint = React.useContext(ActiveBreakpointContext);

  switch (activeBreakpoint) {
    case 'tiny':
    case 'small':
    case 'medium':
      return List;
    case 'large':
    case 'huge':
    case 'enormous':
    default:
      return Table;
  }
};


```

Reply:
Link for components https://github.com/cerner/terra-ui/pull/310/commits/2f0433b8a6f6fb97139694a22a60e0ad1862bf40?file-filters%5B%5D=.jsx&file-filters%5B%5D=.mdx#diff-b887d62719eff551778a1fc191b08382c949a96aa056b14797bdcf836a34f062R9

And swapped to using `ActiveBreakpointContext` for all examples https://github.com/cerner/terra-ui/pull/310/commits/2f0433b8a6f6fb97139694a22a60e0ad1862bf40?file-filters%5B%5D=.jsx&file-filters%5B%5D=.mdx#diff-5bb9914536b23dafae1ea592488ce19fda98bedad1874ac8546a1dedbc6cdd38R188